test/parallel/test-http-request-arguments.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const http = require('http');
+
+// two arguments: URL + options
+const request = http.request('http://example.com', { path: '/path' });
```

Comment:
I think it would be better to have a real test by setting up a local `http.Server` on a random port, connecting to that, and verifying the path and such inside the incoming connection handler (wrapped in a `common.mustCall()`).